examples/rust/revy/README.md

```diff
@@ -0,0 +1,52 @@
+<!--[metadata]
+title = "Revy"
+source = "https://github.com/rerun-io/revy"
+tags = ["2D", "3D", "Integration", "Gamedev"]
```

Comment:
No idea what are the available/allowed tags?

Reply:
Anything goes for now.

"Bevy" would be nice tag though, since only the title and the tags show up on the example page.
Or title it "Revy - Rerun integration for Bevy" or domething